gpkit/model.py

```diff
@@ -300,8 +300,7 @@ def _solve(self, programType, solver, verbosity, skipfailures,
                            for (var, grid) in zip(sweep, sweep_grids)}
 
             if verbosity > 0:
-                print("Solving for %i variables over %i passes." %
-                      (len(self.variables), N_passes))
+                print("Solving over %i passes." % N_passes)
```

Comment:
@bqpd please confirm you agree with the message change -- @pgkirsch mentioned in the pull request comments that the # variables message gets repeated by the GP solve.